microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -1003,21 +1124,24 @@ See also:
 * [Application tool, to manage storage](../storage/application-tool.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is not present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
 
 
 <!-- ====================================================================== -->
-## Enable the legacy tracing model when inspecting Chrome traces using the performance panel
-<!-- no (?) link -->
+## Enable the experimental, WIP sidebar for the Performance Panel.
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+<!-- checkbox label has period -->
 
-_new as of Microsoft Edge 126_
+_new as of Microsoft Edge 128_
 
-An internal setting that controls whether to enable the legacy tracing model that the **Performance** tool uses.  Do not use.
+In the **Performance** tool, enables the Windows Information Protection (WIP) sidebar.
+<!-- todo: real desc - check article -->
```

Comment:
Enable the experimental, WIP sidebar for the Performance Panel.
need real desc @captainbrosset - expand "WIP" correctly.
check https://learn.microsoft.com/en-us/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip

Reply:
This experiment is a work in progress (WIP). Enabling it adds a vertical sidebar to the **Performance** tool. The sidebar will later contain performance insights helping you find bottlenecks in your recorded performance traces more quickly.

Reply:
Let's not add any see also here, I can't think of any that would work.

Reply:
```
Enables the sidebar in the **Performance** tool.  The sidebar is an experimental work in progress (WIP).

<!-- omit See Also -->

<!-- todo: screenshot of resulting UI, eg in Canary -->
```
@captainbrosset fyi i'm not seeing the resulting sidebar in the ui, so no screenshot